### ISSUE TITLE
Simple remoting example for latest version

### ISSUE THREAD

**Issue by Ulfasaar** [HaxeFoundation/haxe#6733]
Hi guys,

First off great job with haxe so far, I think its absolutely amazing :)

I am currently in the process of trying to port the IOT system I programmed for Massey University in New Zealand to using haxe, instead of just using raw Python, C++, Javascript and PHP.

The first thing I want to do with haxe is make a small subsystem that removes outliers in the data that is sent to clients.

I have already written the predictive models for this in Python and pickled them. I also have a nodejs server that is currently receiving data from the sensor systems, that also allows the clients to download data.

As a result I would like to use haxe to program a small python script for the server that uses those pickled models to process data that is sent to it from the server, and then return its output.

I already made the code for pulling in the models using python externs and the joblib library however I am struggling to get the connection to work.

I think that for my use case I just want to use a haxe.remoting.LocalConnection, as both systems will be on the same server, but I am struggling to find up to date documentation and examples online for how to do so.

If someone can point me in the right direction, or provide a really barebones example of some client server code for haxe version 3.4.4 that would be much appreciated.

PS: I am happy to help with fleshing out documentation,  especially because I am hoping to use haxe a lot more going forward :)

**Comment by Ulfasaar** [comment]
Also I have a suggestion for the visual studio code language server, I don't know if it counts as a feature request however, and if it does where the correct place to talk about it is so if someone can tell me that too that would be fantastic

**Comment by back2dos** [comment]
As for the latter, try https://github.com/vshaxe/vshaxe ;)

**Comment by Ulfasaar** [comment]
Thanks back2dos, 

worked out that my request was just a normal issue and posted it in vshaxe. When I say issue by the way it was more just me forgetting that vscode extensions generally don't like it when the stuff you want to work on isn't in the root of the workspace :p

**Comment by markknol** [comment]
Doesn't this help? It should work:
https://haxe.org/manual/std-remoting-connection.html

server:
```haxe
class Server {
  function new() { }
  function foo(x, y) { return x + y; }

  static function main() {
    var ctx = new haxe.remoting.Context();
    ctx.addObject("Server", new Server());

    if(haxe.remoting.HttpConnection.handleRequest(ctx))
    {
      return;
    }

    // handle normal request
    trace("This is a remoting server !");
  } 
}
```

client:
```haxe 
class Client {
  static function main() {
    var cnx = haxe.remoting.HttpAsyncConnection.urlConnect("http://localhost/");
    cnx.setErrorHandler( function(err) trace('Error: $err'); } );
    cnx.Server.foo.call([1,2], function(data) trace('Result: $data'););
  }
}
```

**Comment by Ulfasaar** [comment]
Thanks markknol,

I finally got a chance to try the example you posted. The thing that confused me before was the context. 
I think I get it now. Am I right to think that it's a way for us to declare which objects can have their functions called remotely by the client? 

Also there were a few syntax errors in the client code that I cleaned up quickly. Mostly just missing curly braces on the fourth and fifth lines.

I am also running IIS on the same machine and I was getting a error that said
 **Client.hx:4: Error: Http Error #502** before I stopped it.

It now returns a new error that says
**Client.hx:4: Error: Failed to connect on 127.0.0.1:80**

I haven't worked out why just yet but my best guess is that something is using that port. It doesn't seem like quitting docker has affected it either.

How do I change the port it is listening on to port 81 for instance? Just because I often have a issue with using port 80 on this machine for some reason.

Thanks and kind regards,
Ryan Weyers
